ci/jobs/scripts/clickhouse_proc.py

```diff
@@ -347,7 +347,17 @@ def start_minio(self, test_type):
                 command, stdout=log_file, stderr=subprocess.STDOUT
             )
         print(f"Started setup_minio.sh asynchronously with PID {self.minio_proc.pid}")
-        return True
+
+        for _ in range(20):
+            res = Shell.check(
+                "/mc ls clickminio/test | head -n5",
```

Comment:
```suggestion
                "/mc ls clickminio/test",
```

This may work incorrectly, since by default (and this what will happen here likely) the shell will fail only if the last command from it fails. Take a look at https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#set--o-pipefail (or `man bash`)

Reply:
right, i changed to grep -q. to not flood the log